packages/core/bootstrap/src/lib/cache/index.ts

```diff
@@ -185,14 +199,19 @@ export const withCache: Middleware = async (execute, options = defaultOptions())
 
     const result = await execute(data)
     await _cacheOnSuccess(result)
-    return result
+    const debug = {
+      staleness: 0,
+      performance: endMetrics(false, 0),
+      providerCost: result.data.cost || 1,
+    }
+    return { ...result, debug }
   }
 
   // Middleware wrapped execute fn which cleans up after
   return async (input) => {
     const result = await _executeWithCache(input)
     // Clean the connection
-    await cache.close()
+    // await cache.close()
```

Comment:
Should we get rid of this @krebernisak?

Reply:
Let's sync offline on that! I've noticed this comment but thought you were experimenting with something.

We need to close the connection after ourselves, or we are just going to open new connections indefinitely and slow down the Redis node or make it crash. So now I'm interested in how this line affects the warmer.